index.css

```diff
@@ -0,0 +1,158 @@
+@import url(https://fonts.googleapis.com/css?family=Playfair+Display|Poppins);
+
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  width: 100vw;
+  overflow-x: hidden;
+}
+
+a {
+  text-decoration: none;
+}
+
+nav a {
+  display: none;
+}
+
+header > * {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.single-shape .triangle {
+  background-image: url(images/triangle.png);
+  height: 100%;
+  width: 100%;
+  background-repeat: no-repeat;
+}

```

Comment:
- [ ] This Should be a logo with `a`  tag inside the html as the requirement says. [see here](https://user-images.githubusercontent.com/61562626/129068415-6c963a6a-3085-4dc8-95e3-3203026e1aa6.jpeg)

Reply:
Not well solved.